classes/migration/upgrade/v3_4_0/I11125_UpdateEmailTemplateVariables.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/v3_4_0/I11125_UpdateEmailTemplateVariables.php
+ *
+ * Copyright (c) 2014-2025 Simon Fraser University
+ * Copyright (c) 2000-2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class I11125_UpdateEmailTemplateVariables
+ *
+ * @brief Migration to update Email Template variable names
+ */
+
+namespace PKP\migration\upgrade\v3_4_0;
+
+use Illuminate\Support\Facades\DB;
+
+abstract class I11125_UpdateEmailTemplateVariables extends \PKP\migration\Migration
+{
+    public function up(): void
+    {
+        // Update template variables
+        $this->renameTemplateVariables($this->oldToNewVariablesMap());
+    }
+
+    public function down(): void
+    {
+        $newToOldVariableMap = array_map(function ($variablesMap) {
+            return array_flip($variablesMap);
+        }, $this->oldToNewVariablesMap());
+
+        $this->renameTemplateVariables($newToOldVariableMap);
```

Comment:
Just a note... At the moment, given the number of migrations without a `down()`, I think it's a bit pointless to code it, we need to have a well defined policy. I had started a discussion here: https://github.com/pkp/pkp-lib/discussions/9061